### PR TITLE
EWPP-1911: PHP8 compatibility.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -70,6 +73,13 @@
...
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/installers": true,
+            "drupal/core-composer-scaffold": true,
+            "cweagans/composer-patches": true,
+            "phpro/grumphp": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
```
Comment (nagyad) [inline composer.json]: Should be in alphabetic order.

**Review by nagyad — COMMENTED** [review]
.info.yml needs to be updated too. Reopening.

File: media_avportal.info.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: 'Media AV Portal'
 description: 'Media AV Portal adds European Audiovisual Services as a supported media provider.'
 type: module
 package: Media
-core_version_requirement: ^8.9 || ^9
+core_version_requirement: ^9.2
```
Comment (22Alexandra) [inline media_avportal.info.yml:5]: Please update test modules too.

File: .drone.yml [hunk 3]
```diff
@@ -54,6 +53,7 @@ pipeline:
     volumes:
     - /cache:/cache
     commands:
+    - composer self-update --2
```
Comment (22Alexandra) [inline .drone.yml:56]: We can remove the `--no-suggest`

File: composer.json [hunk 4]
```diff
@@ -6,24 +6,26 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/remote_stream_wrapper": "^1.5",
         "symfony/mime": "^4.3|^5.0"
```
Comment (22Alexandra) [inline composer.json]: We can remove this and its readme entry.

File: composer.json [hunk 5]
```diff
@@ -6,24 +6,26 @@
...
-        "nikic/php-parser": "^3.1.5",
-        "openeuropa/code-review": "~1.6",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
-        "phpunit/phpunit": "^7 || ^8"
+        "nikic/php-parser": "^4.12.0",
+        "openeuropa/code-review": "^2.0",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
+        "phpspec/prophecy-phpunit": "^2",
+        "phpunit/phpunit": "^9.4"
```
Comment (22Alexandra) [inline composer.json]: We don't need this.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Pull request merged**